CHANGES.txt

```diff
@@ -1,4 +1,5 @@
 Current
+Fixed: GITHUB-2080: Wrong text for assertTrue
```

Comment:
You might want to add your name along with the bug information.

Reply:
Hi Krishnan, I have obviously ignored the names that other contributors wrote in the CHANGE.txt, thanks for reminding!
In addition, could you help to figure out why the Travis build failed? I found that it failed also for the two previous commits before mine...